ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessCategoryService.scala

```diff
@@ -0,0 +1,32 @@
+package pl.touk.nussknacker.ui.process
+
+import java.util.Collections
+
+import com.typesafe.config.{Config, ConfigFactory}
+import pl.touk.nussknacker.engine.ProcessingTypeData.ProcessingType
+
+import scala.collection.JavaConverters._
+import net.ceedubs.ficus.Ficus._
+
+trait ProcessCategoryService {
+  def getTypeForCategory(category: String) : Option[ProcessingType]
+  def getAllCategories : List[ProcessingType]
+}
+
+class ConfigProcessCategoryService(config: Config) extends ProcessCategoryService {
+
+  private val defaultCategoriesConfig = ConfigFactory.parseMap(Collections.singletonMap("Default", "streaming"))
```

Comment:
I wonder, maybe we can move it to defaultUiConfig, so we have defaults in one place?